modules/API/MotorController.h

```diff
@@ -0,0 +1,159 @@
+/**
+ * @file MotorController.h
+ * @brief モータ制御に用いる関数をまとめたラッパークラス
+ * @author nishijima515
+ */
+#ifndef MOTOR_MotorController_H
+#define MOTOR_MotorController_H
```

Comment:
インクルードガードが小文字混ざってしまってる。普通に変になってるか。
```suggestion
#ifndef MOTOR_CONTROLLER_H
#define MOTOR_CONTROLLER_H
```